consensus/types/src/payload.rs

```diff
@@ -955,3 +961,84 @@ impl<T: EthSpec> From<BlindedPayload<T>> for ExecutionPayloadHeader<T> {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(untagged)]
+#[serde(bound = "E: EthSpec")]
+pub enum FullPayloadContents<E: EthSpec> {
```

Comment:
This is the response type from the builder `submitBlindedBlock` endpoint, so I think this is probably the best place for it? (same crate as `BuilderBid`)

Reply:
Seems good for now, maybe in the future we'll want to separate out all the builder types. Or maybe ePBS means we need similar here anyways.